tests/test_tokens_prices.py

```diff
@@ -0,0 +1,156 @@
+import setup
+import pytest
+import timeit
+import time
+import os
+
+from datetime import datetime
+from sqs_service import *
+from coingecko_service import *
+from conftest import SERVICE_MAP
+from conftest import SERVICE_COINGECKO
+from constants import *
+from filelock import FileLock
+
+counter_file = "/tmp/counter.txt"
+lock_file = "/tmp/counter.lock"
+
+class TestTokensPrices:
+
+    # Function to read the current counter value
+    def read_counter(self):
+        if not os.path.exists(counter_file):
+            return 0
+        with open(counter_file, "r") as file:
+            return int(file.read().strip())
+
+    # Function to write the new counter value
+    def write_counter(self, value):
+        with open(counter_file, "w") as file:
+            file.write(str(value))
+
+    # Function to safely increment the counter
+    def increment_counter(self):
+        with FileLock(lock_file):
+            counter = self.read_counter()
+            counter += 1
+            self.write_counter(counter)
+
+    # Initialize the counter file at the beginning of the test
+    def setup_method(self):
+        if os.path.exists(counter_file):
+            os.remove(counter_file)
+        self.write_counter(0)
+
+    # Assert that the unsupported token count is within the threshold
+    # Clean up the counter file at the end of the test
+    def teardown_method(self):
+        unsupported_token_count = self.read_counter()
+        assert unsupported_token_count <= UNSUPPORTED_TOKEN_COUNT_THRESHOLD, f"Unsupported token count: {unsupported_token_count} exceeds threshold {UNSUPPORTED_TOKEN_COUNT_THRESHOLD}"
+        if os.path.exists(counter_file):
+            os.remove(counter_file)
```

Comment:
I belive that `setup_method` runs before every price test and `teardown_method` after every price test.

As an outcome, the `setup_method` resets the counter to zero.